docs/application/tizen-studio/release-notes/2-3-release-notes.md

```diff
@@ -0,0 +1,86 @@
+# Tizen Studio 2.3 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   Tizen Studio
+
+	-   [JAVA 9](https://www.oracle.com/java/java9.html) support is added in Tizen Studio. Users can now download and install Tizen Studio along with JAVA 9.
+-   Base IDE
+	-   Tizen Studio is updated from [Eclipse Mars](https://www.eclipse.org/mars/) to [Eclipse Oxygen](https://projects.eclipse.org/releases/oxygen), to help user with improved performance and bug fixes.
+	-   CDT (C/C++ Development Tooling) is updated to 9.3.2 and the Debugger is migrated from CDI to DSF enabling faster debugging and easier extension.
+-	Emulator Manager
+
+	-   The Emulator Manager is updated to support JAVA 9.
+	-   [Intel� HAXM](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/hardware-accelerated-execution-manager) is upgraded to 7.0.0 to support Tizen Emulators on Windows 10 and Mac.
+-	Device Manager
+	-   The Device Manager is updated to support JAVA 9.
+-	Profiling Tools
+	-   Dynamic Analyzer and TTrace are updated to support JAVA 9.
+-	Installer, Package Manager, and Uninstaller
+	-   Package Manager and Installer are updated to support JAVA 9.
+-	RT IDE (Integrated Development Environment)
+	-   Base IDE of RT IDE is migrated from Eclipse Mars to Eclipse Oxygen, to help user with improved performance and bug fixes.
+### Changed Features
+-	Certificate Manager
+	-   ST (Smart Things) certification is simplified by enabling auto certificate generation using Samsung Account sign-in.
+### Fixed Bugs
+-	Emulators
+	-   Fixed a bug in Emulator Manager that shows the deleted skin in the list of device definition.
+-	Device Manager
+	-   Fixed a bug in Device Manager, now the remote device manager of Tizen Studio and remote device manager of Device Manager are in sync.
+-   Web and Native IDE
+	-   Fixed a bug in common-eplugin, earlier it used to return last connected device even if the device was disconnected.
+### Known Issues
+-	Tizen Studio
+	-	In case of MAC if after tizen studio update the UI perspectives are not displayed properly please restart the IDE. This issue is because of uncleared cache from previous installation.
```

Comment:
If the two sentences given under Tizen Studio are interrelated, please rephrase them as:

o	In case of Mac, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.

o	After updating Tizen Studio and restarting the IDE, if you see some old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.

Reply:
Hi @Reni-90 
These 2 issues are independent. Please let us know if we need to rephrase the same as they are not interrelated.
Thanks

Reply:
If the two issues are independent, please rephrase them as:
o	In case of Mac, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
o	In case if you see some old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.